CRM/Core/Payment/AuthorizeNetIPN.php

```diff
@@ -230,7 +230,9 @@ public function getInput(&$input, &$ids) {
       $input['trxn_id'] = md5(uniqid(rand(), TRUE));
     }
 
-    if (!$this->getBillingID($ids)) {
+    $ids['billing'] = CRM_Core_BAO_LocationType::getBilling();
```

Comment:
@mattwire so what I thought we agreed was to remove 
```
 if (!$ids['billing']) {
      \Civi::log()->error(ts('Please set a location type of %1', [1 => 'Billing']));
      return FALSE;
    }
```

entirely

Reply:
@eileenmcnaughton Ok, see now?